spec/syntax.md

```diff
@@ -345,19 +345,18 @@ Whitespace within a _pattern_ is meaningful and MUST be preserved.
 
 ### Expressions
 
-**_Expressions_** can either start with an operand or a function call.
+_Expressions_ ***must*** start with a _literal_, a _variable_, or an _annotation_, or consist of a _reserved_ string. An _expression_ ***must not*** be empty.
 
-The operand is a literal or a variable name.
-The operand can be optionally followed by an _annotation_:
-a function and its named options.
-Functions do not accept any positional arguments
-other than the operand in front of them.
+A _literal_ or _variable_ ***may*** be optionally followed by an _annotation_. 
 
-Function calls do not require an operand as an argument,
-but an _expression_ must not be completely empty.
+An _annotation_ consists of a _function_ and its named _options_.
+
+_Functions_ do not accept any positional arguments other than the _literal_ or _variable_ in front of them.
+
+_Reserved_ sequences start with a reserved character and are intended for future standardization.
```

Comment:
Should we include some explicit language about whether implementations are allowed to define their own meanings for parts of the reserved space?

Reply:
We should not allow that unless we specifically reserve one or more sigils for private use. It was on my mind writing this to propose private use, but I deferred those thoughts in favor of getting this in.

What we don't want is for vendor foo to use (let's say) `~` for private use and then MF2.1 to adopt `~` for something else. Private agreement is another thing and leaving room for that is probably a Good Idea.

Reply:
I've filed #378 to discuss this separately.